pkg/actuators/machine/utils.go

```diff
@@ -81,8 +82,8 @@ func getInstances(machine *machinev1.Machine, client awsclient.Client, instanceS
 			Values: []*string{&machineName},
 		},
 		{
-			Name:   aws.String("tag:clusterid"),
-			Values: []*string{&clusterID},
+			Name:   aws.String("tag:kubernetes.io/cluster/" + clusterID),
+			Values: []*string{&owned},
 		},
```

Comment:
can we put owned into a constant e.g `ResourceLifecycleOwned`? can we also use `aws.StringSlice` similar to line 262?

Reply:
I tried, was not overly happy about the pointer from the const but I'll try to get it working.